hack/compare-performance.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ "$#" -lt 2 ]]; then
+	echo "Usage: ${0} rev1 rev2 [num-iterations [summary-dir]]
+	To be run from kueue's root dir."
+	exit 1
+fi
+
+REV_1="$1"
+REV_2="$2"
+NUM_IT=${3:-5}
+SUMMARY_DIR=${4:-$(mktemp -d)}
+
+STATS_FILE="${SUMMARY_DIR}/perf_stats.yaml"
+SUMMARY_FILE="${SUMMARY_DIR}/perf_stats.yaml}"
+
+if [ -f "${STATS_FILE}" ]; then
+	rm "${STATS_FILE}" 
+fi
+
+if [ -f "${SUMMARY_FILE}" ]; then
+	rm "${SUMMARY_FILE}" 
+fi
+
+echo "
+Using:
+Stats file: $STATS_FILE
+Summary file: $SUMMARY_FILE
+"
+
+#$1 - iteration
+#$2 - git-rev 
+function run_and_store {
+	git reset --hard 
```

Comment:
Please drop this command, someone may lose its unstaged work by running this script.

Reply:
I genualy don't know which is worst, lose some data or run with uncommitted data.... , maybe we should jus error out if the git tree is not clean.

Reply:
erroring in that case is certainly better

Reply:
done